desc/compute/_profiles.py

```diff
@@ -607,6 +607,97 @@ def _iota_rr(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="iota_current",
+    label="\\iota_{\\mathrm{current}}",
+    units="~",
+    units_long="None",
+    description="Rotational transform (normalized by 2pi), current contribution",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="r",
+    data=["iota_den", "iota_current_num"],
+)
+def _iota_current(params, transforms, profiles, data, **kwargs):
+    data["iota_current"] = transforms["grid"].replace_at_axis(
+        data["iota_current_num"] / data["iota_den"], 0
+    )
+    return data
+
+
+@register_compute_fun(
+    name="iota_vacuum",
+    label="\\iota_{\\mathrm{vacuum}}",
+    units="~",
+    units_long="None",
+    description="Rotational transform (normalized by 2pi), vacuum contribution",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="r",
+    data=["iota_den", "iota_vacuum_num"],
+    axis_limit_data=["iota_den_r", "iota_num_r"],
+)
+def _iota_vacuum(params, transforms, profiles, data, **kwargs):
+    data["iota_vacuum"] = transforms["grid"].replace_at_axis(
+        data["iota_vacuum_num"] / data["iota_den"],
+        lambda: data["iota_num_r"] / data["iota_den_r"],
```

Comment:
Similar to https://github.com/PlasmaControl/DESC/pull/642/files#r1309419402, this one's limit should be `beta_r / iota_den_r` where `beta_r` is the same thing as `iota_vacuum_num_r`.

Even though the enclosed current  `I` is 0 at the magnetic axis, for it to not contribute to the rotational transform at axis it needs to approach 0 fast - in particular fast enough that both $\partial_{\rho} I$ and $\partial_{\rho \rho} I$ are 0 axis. $\partial_{\rho \rho} I$ need not be zero at axis, so both iota_current and iota_vacuum may be nonzero at axis.